packages/core/test/combinator/multicast-test.js

```diff
@@ -0,0 +1,191 @@
+import { describe, it } from 'mocha'
+import { assert, eq } from '@briancavalier/assert'
+import { spy } from 'sinon'
+
+import { now } from '../../src/source/now'
+import { map, tap } from '../../src/combinator/transform'
+import { multicast, MulticastSource, MulticastDisposable } from '../../src/combinator/multicast'
+import { runEffects } from '../../src/runEffects'
+import { ticks, makeEvents, collectEvents } from '../helper/testEnv'
+import { sinkSpy } from '../helper/sinkSpy'
+import FakeDisposeStream from '../helper/FakeDisposeStream'
+
+describe('multicast', () => {
+  it('should be identity for already-multicasted stream', () => {
+    const s = multicast(now(1))
+    eq(s, multicast(s))
+  })
+
+  it('should call mapper function once when there are > 1 observer', () => {
+    const observer1Spy = spy()
+    const observer2Spy = spy()
+    const observer3Spy = spy()
+
+    const f = x => x + 1
+    const x = Math.random()
+    const s = now(x)
+
+    const mapperSpy = spy(f)
+    const mapped = map(mapperSpy, s)
+    const multicasted = multicast(mapped)
+
+    const o1 = runEffects(tap(observer1Spy, multicasted), ticks(1))
```

Comment:
A lot of these tests use separate schedulers now (i.e. Calling ticks multiple times in the same test). The tests pass, but it's not representative of normal/supported use cases, where you'd have exactly 1 scheduler per application.

So, I think the right thing is for each test to create 1 scheduler, like 'should publish events to all observers' does below.